File: SOLID/C#/LSP/LocalUploader.cs
using System;

namespace LSP
{
    class LocalUploader : IFileUploader
    {
        public bool CanUpload(long sizeInMb) => true;

        public void Upload(string fileName, long sizeInMb)
        {
            Console.WriteLine($"Local upload: {fileName}");
        }
    }
}

File: SOLID/TS/LSP/LocalUploader.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.LocalUploader = void 0;
var LocalUploader = /** @class */ (function () {
    function LocalUploader() {
    }
    LocalUploader.prototype.canUpload = function (sizeInMb) {
        return true;
    };
    LocalUploader.prototype.upload = function (fileName, sizeInMb) {
        console.log("Local upload: ".concat(fileName));
    };
    return LocalUploader;
}());
exports.LocalUploader = LocalUploader;
